apps/AgeDetection/agedetection.cpp

```diff
@@ -82,10 +82,10 @@ class AgeDetector {
         std::string age = ageList[maxIndiceAge];
         NDN_LOG_INFO("Age: " << age);
 
-        jsonInput["Age"] = std::stoi(age);
+        jsonInput["Age"] = age;
```

Comment:
Is it intentional that the string-to-integer conversion is removed here?

Reply:
Yes, this is intentional. Here age is not an integer but an age group which is a string.